app/src/ui/dialog/dialog-options.html

```diff
@@ -8,7 +8,7 @@
     <template>
         <style include="shared"></style>
 
-        <pl-dialog open="{{ open }}" prevent-dismiss>
+        <pl-dialog open="{{ open }}" prevent-dismiss$="{{ preventDismiss }}">
```

Comment:
Just a minor thing, but a one-way binding would be better here. I.e. `[[ preventDismiss ]]`.